Documentation/network/clustermesh/gke-clustermesh-prep.rst

```diff
@@ -8,7 +8,7 @@ This is a step-by-step guide on how to install and prepare
 Google Kubernetes Engine (GKE) clusters to meet the requirements 
 for the clustermesh feature.
 
-In this guide we will deploy two zonal, single node GKE clusters
+In this guide you will deploy two zonal, single node GKE clusters
```

Comment:
Or “This guide describes how to deploy...”

Reply:
Done